fire-cakes/pancakes
# Pieces controller

### What does this PR do?
[Support moving pieces. Later we can make the pieces draggable/droppable and remove the interstitial page.]

### Where should I start?
[pieces_controller.rb/view pages]

### How do I manually test this?
[Startup local directory. Login and join a game highlight each piece to see coordinates. Click the on any piece and check URL.]

### Additional Comments
[The chess board looks off right now. Is not a major problem.]

### Trello story
[List or link]

### GIF for how this PR makes me feel
![](GIF_URL)
